ansible/roles/debops.postfix/tasks/ldap.yml

```diff
@@ -0,0 +1,59 @@
+---
+
+- name: Make sure 'postfix/ldap' directory exists
+  file:
+    dest: '/etc/postfix/ldap'
+    state: 'directory'
+    owner: 'root'
+    group: 'root'
```

Comment:
You shouldn't specify owner and group if these are `root:root`, that will be the default result. Doing it this way helps on different operating systems, like FreeBSD, where `root` UNIX group may not exist and `wheel` is usually used instead.